### DISCUSSION TITLE
SDXL Turbo: A Real-Time Text-to-Image Generation Model

### DISCUSSION THREAD

**Discussion by drax-xard** [vladmandic/sdnext#2572] [Ideas]
> Today, we are releasing SDXL Turbo, a new text-to-image mode. SDXL Turbo is based on a novel distillation technique called Adversarial Diffusion Distillation (ADD), which enables the model to synthesize image outputs in a single step and generate real-time text-to-image outputs while maintaining high sampling fidelity. For researchers and enthusiasts interested in technical details, our research paper is available [here](https://stability.ai/research/adversarial-diffusion-distillation). It's important to note that SDXL Turbo is not yet intended for commercial use.

[Blog post]( https://stability.ai/news/stability-ai-sdxl-turbo)
[Paper](https://static1.squarespace.com/static/6213c340453c3f502425776e/t/65663480a92fba51d0e1023f/1701197769659/adversarial_diffusion_distillation.pdf)
[Model weights](https://huggingface.co/stabilityai/sdxl-turbo)

You can try the themo at [Clipdrop](https://clipdrop.co/stable-diffusion-turbo )

**Comment by drax-xard** [comment]
From my very limited testing with clipdrop it is, indeed, fast AF.
The quality is meh, mainly on faces and photographic details, but hopefully fine-tuning can improve that.

**Reply by lxkaka** [reply]
+1

**Comment by ilikenwf** [comment]
Seems that the comfyui guys have a turbo workflow for sd 1.5:

https://old.reddit.com/r/StableDiffusion/comments/187ps59/got_realtime_turbo_workflow_working_sd_15_lcm_and/

**Reply by vladmandic** [reply]
read the thread - that same stuff works here. quality is also improved in dev branch already.

**Reply by ilikenwf** [reply]
Well, I tried, assuming the 1.5 lora here is the one to use...

With LCM do I need models other than normal 1.5 models or is the lora sufficient, because using the guide int he comments my outputs suck.

https://huggingface.co/latent-consistency/lcm-lora-sdv1-5/tree/main

**Reply by vladmandic** [reply]
use dev branch or wait until tomorrow, planned merge dev to master. like i said, quality is improved in dev branch.
